indexer-app/src/cmr/indexer/services/index_service.clj

```diff
@@ -292,77 +294,67 @@
   (try
     (cp/parse-concept context collection)
     (catch Exception e
-      (error (format "An error occurred while parsing collection: %s"
-                     (.getMessage e)))
-      nil)))
+      (error (format "An error occurred while parsing collection for autocomplete with concept-id [%s]: %s"
+                     (:concept-id collection)
+                     (.getMessage e))))))
 
-(defn- collection->suggestion-doc
+(defn- collections->suggestion-docs
   "Convert collection concept metadata to UMM-C and pull facet fields
   to be indexed as autocomplete suggestion doc"
   [context collections provider-id]
-  (let [parse-coll (partial parse-collection context)
+  (let [{:keys [index-names]} (idx-set/get-concept-type-index-names context)
+        index (get-in index-names [:autocomplete :autocomplete])
+        parse-coll-fn (partial parse-collection context)
+        humanize-fields-fn (partial humanizer/collection-humanizers-elastic context)
         parsed-concepts (->> collections
                              (remove :deleted)
-                             (map parse-coll)
+                             (map parse-coll-fn)
```

Comment:
I think I actually prefer just having this line be `(map (partial parse-collection context))`. Your eyes don't have to jump around the screen as much